src/zitadel.rs

```diff
@@ -145,6 +166,25 @@ impl Zitadel {
 
 				if let Err(error) = status {
 					tracing::error!("Failed to update user `{}`: {:?}", new.log_name(), error);
+
+					if error.to_string().contains("invalid UpdateHumanPhoneRequest") {
```

Comment:
We really shouldn't copy-paste code like this.

Reply:
I exported the check to a function.